Animations/Zoom/Up.js

```diff
@@ -26,5 +28,5 @@ var ZoomUp = {
     'transform': 'scale(.1) translateY(20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomUp', ZoomUp);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomUp', ZoomUp);
```

Comment:
This does not need to have `'ZoomUp'` passed into it since we have defined a `className` property in `ZoomUp`.